saleor/order/actions.py

```diff
@@ -208,9 +207,8 @@ def clean_mark_order_as_paid(order: "Order"):
 
 def fulfill_order_line(order_line, quantity):
     """Fulfill order line with given quantity."""
-    country = get_order_country(order_line.order)
     if order_line.variant and order_line.variant.track_inventory:
-        decrease_stock(order_line.variant, country, quantity)
+        decrease_stock(order_line, quantity)
```

Comment:
What if an exception is raised?

Reply:
If this line rise error it's mean that this order hasn't enough allocated items. For a better solution here we should refactor the whole FulfillmentCreate, we should have to fulfill all lines in one transaction instead of doing it line by line.